### PR TITLE
Handle nonexistent timestamps

### PR DESCRIPTION
**Pull request by axelstudios**
#### What's this PR do?
- Handles the corner case where timezone-unaware meter timestamps cannot exist, e.g. during the hour when daylight savings "springs forward" to dst.
- This PR also adds tests for the ambiguous and nonexistent scenarios.

#### How should this be manually tested?
See the related ticket

#### What are the relevant tickets?
#4293

### CODE REVIEW

File: seed/data_importer/meters_parser.py [hunk 1]
```diff
@@ -266,11 +265,18 @@ def _parse_pm_meter_details(self):
             except AmbiguousTimeError:
                 # Handle timestamp that occurs twice due to "falling back" to standard time
                 start_time = make_aware(unaware_start, timezone=self._tz, is_dst=False)
+            except NonExistentTimeError:
```
Comment (nllong) [inline seed/data_importer/meters_parser.py:268]: crazy

**Review by nllong — APPROVED** [review]
love the tests, looks good!
